ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md

```diff
@@ -56,13 +56,13 @@ pod_name: < POD_Name >
 
 The following table provide information on the default node types that have been pre-defined in [`eos_designs/defaults/main/defaults-node-type-keys.yml`](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/roles/eos_designs/defaults). To customize or create new node types, please refer to [node types definition](node-types.md)
 
-| Node Type Key      | Underlay Router | Uplink Type | Default EVPN Role  | L2 Network Services | L3 Network Services | VTEP | MLAG Support | Connected Endpoints |
+| Node Type Key      | Underlay Router | Uplink Type  | Default EVPN Role | L2 Network Services | L3 Network Services | VTEP | MLAG Support | Connected Endpoints |
```

Comment:
Is it really needed to change all this?

Reply:
Please avoid "fixing" unrelated stuff in a feature PR.

Reply:
@ClausHolbechArista @xaviramon this is done by linter that is part of AVD devcontainer. It wasn't a manual change.

Reply:
I am not sure which container or linter that is. We don't have this linting in our CI.
Please avoid adding these changes that are unrelated to the feature.

Reply:
As discussed, we will accept these for now and decide what we will do in the future on the next dev call.